### DISCUSSION TITLE
page.bottom_appbar has no visible property

### DISCUSSION THREAD

**Discussion by syleishere** [flet-dev/flet#3729] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

Setting page.bottom_appbar visible property fails

does not currently work

### Code sample

`page.bottom_appbar.visible = False`

will produce an error

### To reproduce

`page.bottom_appbar.visible = False`

### Expected behavior

`page.bottom_appbar.visible = False`

should work just as

`page.appbar.visible = False`

currently does.

### Screenshots / Videos

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Operating System

macOS

### Operating system details

All OSes

### Flet version

all flet versions

### Regression

No, it isn't

**Comment by ndonkoHenri** [comment]
`BottomAppBar` class has a `visible` property.
Please share some code, for further assistance.

**Comment by syleishere** [comment]
Your right, I am unsure why it was not working for me yesterday when I tried it. My bad, can close this.
